From 0619ecff7b91c83410448a896840dc175d678840 Mon Sep 17 00:00:00 2001
From: David Teigland <teigland@redhat.com>
Date: Fri, 3 Apr 2026 13:15:26 -0500
Subject: [PATCH 4/4] lvmpersist: fix grep warning about quoting

checking for PR keys on an nvme device called grep with unnecessary
quoting, causing the warning: grep: warning: stray \ before :

(cherry picked from commit 091dd5bd0a8acecd734e8ef8c070486b1947ccf0)
---
 scripts/lvmpersist.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/lvmpersist.sh b/scripts/lvmpersist.sh
index 9cee385ed..31ea847b9 100755
--- a/scripts/lvmpersist.sh
+++ b/scripts/lvmpersist.sh
@@ -90,7 +90,7 @@ set_type() {
 key_is_on_device_nvme() {
 	FINDKEY_DEC=$(printf '%u' "$FINDKEY")
 
-	if nvme resv-report --eds -o json "$dev" 2>/dev/null | grep -q "\"rkey\"\:${FINDKEY_DEC}"; then
+	if nvme resv-report --eds -o json "$dev" 2>/dev/null | grep -q '"rkey":${FINDKEY_DEC}'; then
 		true
 		return
 	fi
-- 
2.53.0

